src/forklift/core.py

```diff
@@ -78,6 +78,8 @@ def remove_temp_table(table):
         if not arcpy.Exists(crate.destination):
             log.debug('%s does not exist. creating', crate.destination)
             _create_destination_data(crate)
+            if arcpy.Exists(path.join(hash_gdb_path, crate.name)):
```

Comment:
If the update has happened before, then the destination data gets deleted, the hash will still exist and the destination to hash primary key will be invalid.

This probably shouldn't ever happen. Although, I did it to myself when I was playing with forklift.